cloudfoundry-cli/parts/plugins/x-cf.py

```diff
@@ -0,0 +1,8 @@
+from snapcraft.plugins import go
+
+
+class CFPlugin(go.GoPlugin):
+
+    def _local_build(self):
+        self._run(['go', 'install', './{}/...'.format(
+            'github.com/cloudfoundry/cli/main')])
```

Comment:
ok, you are smart people. Removing sources and adding go-packages works. I haven't thought about it. Of course, then we can't use the tags. This is a bug, right?